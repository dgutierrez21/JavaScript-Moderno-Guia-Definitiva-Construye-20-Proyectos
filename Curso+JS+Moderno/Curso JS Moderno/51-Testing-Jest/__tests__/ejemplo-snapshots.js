const cliente = {
  nombre: "Kendal Martinez",
  balance: 500,
  tipo: "Premium",
};

describe("Testing al cliente", () => {
  test("El nombre es Kendal Garrett", () => {
    expect(cliente).toMatchSnapshot();
  });
});

// npm t -- -u para actualizar un snapshot
