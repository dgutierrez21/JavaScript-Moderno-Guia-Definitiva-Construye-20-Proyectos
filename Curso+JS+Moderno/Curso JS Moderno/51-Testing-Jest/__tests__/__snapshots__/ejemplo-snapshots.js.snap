// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente El nombre es Kendal Garrett 1`] = `
{
  "balance": 500,
  "nombre": "Kendal Martinez",
  "tipo": "Premium",
}
`;
